src/main/java/org/prebid/server/analytics/pubstack/PubstackEventHandler.java

```diff
@@ -0,0 +1,198 @@
+package org.prebid.server.analytics.pubstack;
+
+import com.fasterxml.jackson.databind.node.ObjectNode;
+import io.vertx.core.AsyncResult;
+import io.vertx.core.MultiMap;
+import io.vertx.core.Vertx;
+import io.vertx.core.http.HttpHeaders;
+import io.vertx.core.http.HttpMethod;
+import io.vertx.core.logging.Logger;
+import io.vertx.core.logging.LoggerFactory;
+import org.apache.http.HttpStatus;
```

Comment:
I suggest that we stick with `io.netty.handler.codec.http.HttpResponseStatus` because we already use it throughout our codebase.